### PR TITLE
ATCA Time Scan

### PR DESCRIPTION
**Pull request by rlodico06**
Created new versions of the scan.py script for Zones 2 and 3. The Zone 2 version has been tested and functions as intended. The Zone 3 version will be tested when there is a window of availability. The two script versions are identical with the exception of PV names.
Scan Script Results

### CODE REVIEW

**Comment by rlodico06** [comment]
Recommendations from discussion with Dring and Tyler:
- Spit out error instead of target time and counter time
- Query the counter time while looping to check for any unexpected changes in counter time
- Compare on the phase motor status instead of counter
- Add exception handling for large changes in tabletop counter time

**Comment by junyangx** [comment]
Compare on phase motor status in addition to

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: exp-timing/zone2_scan.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# scanning parameters 
+init_step = 0.05 # initial step size
+stop = 8 # total number of steps to take
+mult = 10 # multiplier by which to increase/decrease step size after each step
+wait_time = 5 # wait time in seconds b/t steps
+
+# time scan
+for x in range(0, stop):
+    if (x<(stop/2)):
+        caput(tgt_pv,tgt+init_step*(mult**x), wait=True) # step up with increasing step size
```
Comment (slactjohnson) [inline exp-timing/zone2_scan.py:26]: This setup is kind of weird. Why are you using `caput` after already creating a `PV` class for the PV? You can just use `tgt_pg.put()`. At the beginning of the script you can instantiate your PV classes and then just use those throughout. 

https://pyepics.github.io/pyepics/pv.html#the-pv-class

**Review by slactjohnson — CHANGES_REQUESTED** [review]
This needs a little work before I would approve a merge. 
Let me know if you have any questions.

**Comment by stephf0716** [comment]
This is not the right repo to put this script into. This is where the femto script for LCLS-I lives. For shared code for our group, please add it to [tss-source](https://github.com/slaclab/tss-source).
I also agree with Tyler's comments, make the changes requested and open a PR in the other repo with the same reviewers.

**Comment by rlodico06** [comment]
I have made a round of changes to address these comments and created [a new PR in the tss-source repo](https://github.com/slaclab/tss-source/pull/2), per Stephanie's request.

**Pull request closed without merging**